src/index.js

```diff
@@ -96,7 +96,11 @@ class Backend {
 
       let ret, err;
       try {
-        ret = JSON.parse(data);
+        if (this.options.parse && typeof this.options.parse === 'function') {
```

Comment:
why not adding `JSON.parse` in getDefaults, so we can always `ret = this.options.parse(data)`?